pkg/apb/ext_creds_test.go

```diff
@@ -7,197 +7,23 @@ import (
 	ft "github.com/openshift/ansible-service-broker/pkg/fusortest"
 )
 
-func TestDecodeOutput(t *testing.T) {
-	output := []byte(`
-Login failed (401 Unauthorized)
-
-PLAY [all] *********************************************************************
-
-TASK [setup] *******************************************************************
-ok: [localhost]
-
-TASK [Bind] ********************************************************************
-changed: [localhost]
-
-TASK [debug] *******************************************************************
-ok: [localhost] => {
-    "msg": "<BIND_CREDENTIALS>eyJkYiI6ICJmdXNvcl9ndWVzdGJvb2tfZGIiLCAidXNlciI6ICJkdWRlcl90d28iLCAicGFzcyI6ICJkb2c4dHdvIn0=</BIND_CREDENTIALS>"
-}
-
-PLAY RECAP *********************************************************************
-localhost                  : ok=3    changed=1    unreachable=0    failed=0
-`)
-	result, err := decodeOutput(output)
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	ft.AssertNotNil(t, result, "result")
-	ft.AssertEqual(t, result["db"], "fusor_guestbook_db", "db is not fusor_guestbook_db")
-	ft.AssertEqual(t, result["user"], "duder_two", "user is not duder_two")
-	ft.AssertEqual(t, result["pass"], "dog8two", "password is not dog8two")
-}
-
-func TestImageCantbePulled(t *testing.T) {
-	output := []byte(`
-Error from server (BadRequest): container "aa-425ef090-5f6f-4a0a-87ed-b072881d944d" in pod "aa-425ef090-5f6f-4a0a-87ed
--b072881d944d" is waiting to start: image can't be pulled
-`)
-	result, err := decodeOutput(output)
-	if err == nil {
-		t.Fatal("decode should've returned an error")
-	}
-
-	if result != nil {
-		t.Fatal("result should've been nil")
-	}
-
-	assertError(t, err, "image can't be pulled")
-}
-
-func TestFailedMessageDecodeOutput(t *testing.T) {
-	output := []byte(`
-PLAY [Deploy rds-apb to openshift] *********************************************
-TASK [setup] *******************************************************************
-ok: [localhost]
-TASK [rds-apb-openshift : set_fact] ********************************************
-ok: [localhost]
-TASK [rds-apb-openshift : rds] *************************************************
-fatal: [localhost]: FAILED! => {"changed": false, "failed": true, "msg": "Region not specified. Unable to determine re
-gion from EC2_REGION."}
-        to retry, use: --limit @/opt/ansibleapp/actions/provision.retry
-PLAY RECAP *********************************************************************
-localhost                  : ok=2    changed=0    unreachable=0    failed=1
-`)
-	result, err := decodeOutput(output)
```

Comment:
Since we don't trail the logs anymore, any error will cause monitor output to exit and decode output will never be called.  So all the testing for decode output isn't valid.